lib/knapsack_pro/config/ci/circle.rb

```diff
@@ -23,8 +23,8 @@ def branch
         end
 
         def project_dir
-          project_repo_name = ENV['CIRCLE_PROJECT_REPONAME']
```

Comment:
Determine project directory with `CIRCLE_PROJECT_REPONAME` was for CircleCI 1.0.

CircleCI 1.0 is deprecated since 2019 so this code is not needed.